src/components/form/input.tsx

```diff
@@ -247,7 +249,7 @@ const Input = ({
                     error={error}
                     disabled={disabled}
                     height={height}
-                    showLabel={label}
+                    showLabel={label === '' ? label : localize(label)}
```

Comment:
```suggestion
                    showLabel={label}
```

Reply:
shouldn't i need to localize the label since its a TString?